11ty/index.njk

```diff
@@ -16,9 +16,17 @@ layout: layouts/index.njk
       <div class="box">
         <p class="subtitle">Ways to help</p>
         <ol class="help">
-          <li> Submit words and definitions through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull requests</a>.</li>
+          <li>Submit words and definitions through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull requests</a> or <a href="https://github.com/tatianamac/selfdefined/issues" rel="noreferral">issues</a>.</li>
           <li>Sponsor this work through <a href="https://github.com/sponsors/tatianamac">GitHub Sponsors</a>.</li>
-          <li>Volunteer writing, design, dev help by <a href="http://www.twitter.com/tatianatmac">DMing me @tatianatmac on Twitter</a>.</li>
+          <li>Volunteer writing, design, and dev help by <a href="https://www.twitter.com/selfdefinedapp">reaching out to @SelfDefinedApp on Twitter</a>.</li>
+        </ol>
+      </div>
+      <div class="box">
+        <p class="subtitle">Resources</p>
+        <ol class="help">
+          <li><a href="/documentation/">Documentation</a></li>
+          <li><a href="https://github.com/tatianamac/selfdefined#-self-defined-dictionary" rel="noreferral">README</a></li>
+          <li><a href="https://github.com/tatianamac/selfdefined/blob/master/CONTRIBUTING.md" rel="noreferral">Contributing Guide</a></li>
         </ol>
```

Comment:
Add documentation?
https://www.selfdefined.app/documentation/

Reply:
I thought relative link `<a href="/documentation/">Documentation</a>` should work. No? I'll change it then.

Reply:
A relative link will work and is the more flexible solution. 

See: https://deploy-preview-93--selfdefined.netlify.com/

Reply:
Yes, correct; I was copying and pasting the full link from the browser and wasn't thinking clearly.